<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>#@cr = ©, #@tm = ®, #@permille = ‰, #@eur = €, #@rub = ₽</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>401</width>
      <height>192</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>C++</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/cpp.jpg</normaloff>:/images/cpp.jpg</iconset>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Python</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/python.jpg</normaloff>:/images/python.jpg</iconset>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Java</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/java.jpg</normaloff>:/images/java.jpg</iconset>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C#</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/c#.png</normaloff>:/images/c#.png</iconset>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PHP</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/php1.jpg</normaloff>:/images/php1.jpg</iconset>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JavaScript</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/javasvript.jpg</normaloff>:/images/javasvript.jpg</iconset>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>110</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>140</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Icon mode</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>531</width>
      <height>191</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>№</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Имя компьютера</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IP адрес</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MAC адрес</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>1</string>
     </property>
     <property name="flags">
      <set>ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>Desktop</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>192.168.0.25</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>00:01:2E:81:62:D1</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>Notebook</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>192.168.0.12</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>4C:CC:6A:B6:57:11</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>Android Phone</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>192.168.0.150</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>1C:1B:0D:2E:CC:21</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>Iphone 13ss</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>192.168.0.29</string>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string>4A:1B:2D:4E:CC:21</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
